docs/environment-setup.md

```diff
@@ -137,67 +137,76 @@ _NOTE: This will not work on WSL (Windows Subsystem for Linux). Use the approach
 
 ## After installing the prerequisites:
 
-In the root directory, run `cp env.example .env` on Linux/Mac or `copy env.example .env`.
-
-_Note: The `env.example` file has examples commented on top of each variable_
-
-### Now you can start Redis and Elasticsearch without errors using one of the following methods:
-
-_Note: Make sure only one of the below options are used when starting Telescope_
-
-#### Option 1: Using Docker
+### Start Docker
 
 ```
 sudo systemctl start docker
-docker-compose up --build redis elasticsearch
 ```
 
 _Note: You may need to add your user to the docker group in Linux to use `docker-compose` without `sudo`. To do this, try `groups $USER` in a terminal and check if docker is in the list of groups. If not, add it with `usermod -aG docker $USER` and reboot._
 
 **Important:** Docker builds Telescope's dependencies at launch and keeps them on disk. In some cases, Docker might try to reuse already-built dependencies or cached data, causing misleading results when testing Telescope. To avoid this, it is recommended to use the command `docker system prune -af --volumes` to remove all already-built Telescope dependencies and ensure fresh deployments.
 More information about docker: [images vs containers](https://www.baeldung.com/docker-images-vs-containers) and [volumes](https://docs.docker.com/storage/volumes/).
 
-#### Option 2: Natively installed:
+### Start telescope
+
+There are different ways to run the application. By default, `.env` will be read; however, if there is no `.env` file in the root, [env.development](../config/env.development) will be used.
```

Comment:
I think the `../config/env.development` link won't work, will it? Should we use the link to the master branch instead or just add the path as text, i.e. just `../config/env.development`?

Reply:
What @yuanLeeMidori has is right, it works this way.

Reply:
Saying that `.env` will be used is a bit misleading.  We only use `.env` for the old back-end, and for the way the front-end reads a few URLs. We're trying to remove the need for an `.env` file, but still support it in some limited ways (e.g. for frontend overrides).